static/styles/app_components.scss

```diff
@@ -248,6 +237,17 @@
     border-color: hsl(0, 0%, 87%);
 }
 
+.informational-overlays .tab-switcher {
+    display: flex;
+}
+
+.informational-overlays .tab-switcher .ind-tab {
+    display: flex;
+    text-overflow: initial;
+    white-space: initial;
+    vertical-align: middle;
+}
+
```

Comment:
It's possible this content should just be moved to `static/styles/informational-overlays.scss`?

Reply:
(Or possible we should turn this into a class that we just add to that particular use place with a name like `tab-switcher-allow-overflow` and then this can all be under `.tab-switcher .allow-overflow`; that is almost certainly cleaner.